src/translations/sqlb_nl.ts

```diff
@@ -0,0 +1,7344 @@
+        <translation>Er is tenminste een record waarin de waarde van dit veld geen geheel getal is. Dit maakt het onmogelijk om de AO-optie toe te passen. Pas de tabelgegevens eerst aan.</translation>
+    </message>
+    <message>
+        <location filename="../EditTableDialog.cpp" line="695"/>
+        <source>Column &apos;%1&apos; has duplicate data.
+</source>
+        <translation>Kolom &apos;%1&apos; heeft gedupliceerde waarden.
+</translation>
+    </message>
+    <message>
+        <location filename="../EditTableDialog.cpp" line="696"/>
+        <source>This makes it impossible to enable the &apos;Unique&apos; flag. Please remove the duplicate data, which will allow the &apos;Unique&apos; flag to then be enabled.</source>
+        <translation>Dit maakt het onmogelijk om de Uniek-optie toe te passen. Verwijder eerst de gedupliceerde waarden, zodat de Uniek-optie toe kan worden gepast.</translation>
+    </message>
+    <message>
+        <location filename="../EditTableDialog.cpp" line="834"/>
+        <source>Are you sure you want to delete the field &apos;%1&apos;?
+All data currently stored in this field will be lost.</source>
+        <translation>Weet je zeker dat je het veld &apos;%1&apos; wilt verwijderen?
+Alle waarden die momenteel opgeslagen zijn in dit veld zullen verloren gaan.</translation>
+    </message>
+    <message>
+        <location filename="../EditTableDialog.cpp" line="957"/>
+        <source>Please add a field which meets the following criteria before setting the without rowid flag:
+ - Primary key flag set
+ - Auto increment disabled</source>
+        <translation>Voeg eerste een veld toe dat aan de volgende criteria voldoet, voordat je de &apos;Zonder rowid&apos; optie toepast:
+ - Primaire sleutel ingeschakeld
+ - Automatisch ophogen uitgeschakeld</translation>
+    </message>
+</context>
+<context>
+    <name>ExportDataDialog</name>
+    <message>
+        <location filename="../ExportDataDialog.ui" line="14"/>
+        <source>Export data as CSV</source>
+        <translation>Gegevens exporteren als CSV</translation>
+    </message>
+    <message>
+        <location filename="../ExportDataDialog.ui" line="22"/>
+        <source>Tab&amp;le(s)</source>
+        <translation>&amp;Tabel(-len)</translation>
+    </message>
+    <message>
+        <location filename="../ExportDataDialog.ui" line="57"/>
+        <source>Colu&amp;mn names in first line</source>
+        <translation>&amp;Kolomnamen op eerste regel</translation>
+    </message>
+    <message>
+        <location filename="../ExportDataDialog.ui" line="77"/>
+        <source>Fie&amp;ld separator</source>
+        <translation>&amp;Veldscheidingsteken</translation>
+    </message>
+    <message>
+        <location filename="../ExportDataDialog.ui" line="102"/>
+        <source>,</source>
+        <translation></translation>
```

Comment:
I think it's a symbol that doesn't need translation, so I think you've left the translation blank.
I also have a similar case in the past, but there was an comments that it is safer to copy the original text for symbol
that do not require translation. Please refer to [here](https://github.com/sqlitebrowser/sqlitebrowser/pull/2329#issuecomment-658202890) for details.